app/controllers/admin/categories_controller.rb

```diff
@@ -49,10 +55,11 @@ def update
     response = { errors: '', success: ''}
     start = Time.now
     begin
-      category = LinkedData::Client::Models::Category.find_by_acronym(params[:id]).first
+      category = LinkedData::Client::Models::Category.find_by_acronym(params[:id], include:'name,acronym,created,description,parentCategory,ontologies' ).first
```

Comment:
extract `'name,acronym,created,description,parentCategory,ontologies'` into a constant a re-use it in the line 106